Content.Shared/Chemistry/Components/Solution.cs

```diff
@@ -639,6 +639,62 @@ public Solution SplitSolutionWithOnly(FixedPoint2 toTake, params string[] includ
             return sol;
         }
 
+        // Floof Start - Make Cryotubes split chemicals evenly and exactly 0.5.
+        /// <summary>
+        /// splits the solution taking up to the specified amount of each reagent from the solution.
+        /// If the solution has less of a reagent than the specified amount, it will take all of that reagent.
+        /// </summary>
+        /// <param name="toTakePer">How much of each reagent to take</param>
+        /// <returns>a new solution containing the reagents taken from the original solution</returns>
+        public Solution SplitSolutionReagentsEvenly(FixedPoint2 toTakePer)
```

Comment:
Pull this out of Upstream and into a _DV system

Reply:
On the system you make, add a static helper function that takes two arguments, instead of a class member that takes 1.